components/switch/README.md

```diff
@@ -90,3 +90,21 @@ It consists of 3 states: On ("Visible"), Off ("Hidden"), and On with Conditions
 - Optional default slot content - Content that will be displayed within the "conditions" opener tooltip when the switch is on.
   
 <!-- docs: end hidden content -->
+
+### Visibility Switch with Conditions
+It may be helpful, depending on the use case, to include additional information in the label, relating to other factors that may affect the availability that the Visibility Switch represents.
```

Comment:
Let me know if this makes sense or not.

Reply:
Maybe splitting this into two separate sentences would make it flow a bit smoother?
e.g. "You may need to have visibility be dependent on additional conditions being met outside of the Visibility Switch. You can include additional information ..."